packages/fluentui/react-icons-northstar/src/components/ContentSettingsIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const ContentSettingsIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg
+      role="presentation"
+      focusable="false"
+      viewBox="2 2 16 16"
+      style={{ overflow: 'visible' }}
```

Comment:
@YuanboXue-Amber  this is the overflow you were talking about, should we allow these icons to go into master in the meantime ?